web/themes/contrib/civictheme/civictheme.post_update.php

```
@@ -864,3 +864,51 @@ function civictheme_post_update_update_editor_allowed_field(): string {
   }
   return (string) new TranslatableMarkup('Allowed tags setting not set, aborting update.');
 }
+
+/**
+ * Add civictheme_fast_fact_card paragraph type and enable it for manual list.
+ */
+function civictheme_post_update_add_fast_fact_card_paragraph(array &$sandbox): ?string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$sandbox'. (Unused Code Rules)

(UnusedFormalParameter)